### ISSUE TITLE
cannot complete long list of dependencies.  Pillow will not install.

### ISSUE THREAD

**Issue by cypherstream** [cmnybo/nrsc5-gui#11]
python3 -m pip install --upgrade Pillow
Collecting Pillow
  Using cached Pillow-7.1.2.tar.gz (38.9 MB)
Building wheels for collected packages: Pillow
  Building wheel for Pillow (setup.py) ... error
  ERROR: Command errored out with exit status 1:
 Complete output (155 lines):
  'gcc' is not recognized as an internal or external command,
  operable program or batch file.

**Comment by cmnybo** [comment]
You need to install gcc and add its install location to whatever the equivalent of $PATH is on windows.
Sorry, I can't help with building an executable. I don't use windows.

**Comment by cypherstream** [comment]
Yeah if it was Ubuntu/ Debian based I would easily get it with apt-get install.  Shoehorning this *nix based environment in Windows is foreign to me.  I think I’ll spin up a VM tomorrow and pass through usb.
